.gitignore

```diff
@@ -35,6 +35,7 @@ timeoverflow_development
 timeoverflow_test
 
 .env
+config/database.yml
```

Comment:
We should decide what we want to use and try to stick to it, because everybody can come up with a slightly different flavor. Furthermore we should use what we decide to use for the development environment for TO, that's almost complete BTW. The trade off is between what we want in production and in the official development environment, and make that decision as customizable as possible using environment variables.